apps/omsorgspengesoknad/src/app/søknad/steps/om-barnet/omBarnetStepUtils.tsx

```diff
@@ -113,3 +114,23 @@ export const mapBarnTilRadioProps = (barn: RegistrertBarn, disabled?: boolean):
         disabled,
     };
 };
+export const barnet18årOgKanFortsette = (
```

Comment:
Selve sjekken på alder kan vel forenkles slik at den kun sjekker ett barn, da slipper en å sende inn hvilket barn det gjelder? Evt. skille det ut selve sjekken på alder i en egen funksjon som kun sjekker på om det er 18 år mellom to datoer?
Er det noen grunn til at du sender inn dagens dato, og ikke bare henter den ut i funksjonen?